src/Settings/SettingsPage.js

```diff
@@ -4,6 +4,7 @@ import { FormattedMessage } from 'react-intl';
 import { Settings } from '@folio/stripes/smart-components';
 
 import CategorySettings from './CategorySettings';
+import TypeSettings from './TypeSettings/TypeSettings';
```

Comment:
please add `index.js` for exporting TypeSettings

Reply:
added index.js

Reply:
it's still not updated, please fix

Reply:
it should be `import { TypeSettings } from './TypeSettings';` after your changes in export

Reply:
sorry, my mistake :/
should be fixed now